src/Microsoft.AspNetCore.Mvc.Formatters.Xml/ProblemDetailsWrapper.cs

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace Microsoft.AspNetCore.Mvc.Formatters.Xml
+{
+    /// <summary>
+    /// Wrapper class for <see cref="Mvc.ProblemDetails"/> to enable it to be serialized by the xml formatters.
+    /// </summary>
+    [XmlRoot(nameof(ProblemDetails))]
+    public sealed class ProblemDetailsWrapper : IXmlSerializable, IUnwrappable
```

Comment:
Does this class need to be `sealed` for some reason? If not, could derive `ValidationProblemDetailsWrapper` from this and have it override `virtual` `ReadProperty(...)` and `WriteProblemDetails(...)` with just a few lines of code each. I realize class size isn't the goal here but repeating the boilerplate is a bit icky.